settings.gradle

```diff
@@ -26,7 +26,7 @@ String[] mods = [
         // new web projects; these modules are intended to form a complete, modular web application,
         // with heavy dependency isolation that should enable very fast rebuilds.
 String[] webMods = [
-        'web',
+        'web-js',
```

Comment:
why rename this (here and elsewhere) and make a new project, directory? This is the aggregation of the public artifacts produced by child projects - should "web" have other meaning than this?

rename aside, this isn't just js, but the html/css to run the page, and html samples to show the api

Reply:
Previously we were targetting the `web` `jsJar` configuration, so I figured the descriptor `js` was appropriate. If we think `web` suffices, or there's a better name, happy to change it.

Reply:
I'd lean towards "no rename", strictly for the sake of not moving files unless needed, but it really doesn't matter.

Reply:
Undid rename.